extensions/blocks/mailchimp/editor.scss

```diff
@@ -2,6 +2,10 @@
 
 .wp-block-jetpack-mailchimp {
 
+	.components-placeholder__label svg {
+		margin-right: 1ch;
```

Comment:
We should fix this in core

Reply:
Ditto to above. Let's keep this hack in here until this is resolved in core.